proto/supply/packages/lodging_package.proto

```diff
@@ -4,9 +4,13 @@ package holms.types.supply.packages;
 option csharp_namespace = "HOLMS.Types.Supply.Packages";
 option go_package = "supply/packages";
 
+import "google/protobuf/timestamp.proto";
 import "supply/priced_offered_lodging.proto";
+import "supply/packages/lodging_package_indicator.proto";
 
 message LodgingPackage {
 	bool bookable = 1;
 	repeated holms.types.supply.PricedOfferedLodging offers = 2;
+	.holms.types.supply.packages.LodgingPackageIndicator package_id = 3;
```

Comment:
Would it make sense for this to also implement EntityDTO? EntityDTO is used to indicate proto classes that have an indicator as a primary key.

Optional, depending on if you think it makes sense.